api/src/main/java/io/strimzi/api/kafka/model/kafka/entityoperator/EntityTopicOperatorSpec.java

```diff
@@ -79,16 +83,30 @@ public void setImage(String image) {
         this.image = image;
     }
 
-    @Description("Interval between periodic reconciliations.")
+    @Description("Interval between periodic reconciliations in seconds.")
```

Comment:
Maybe you can clarify that it will be ignored if `reconciliationIntervalMs` is set?